src/utils/escape-regex.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
@nfantone could you also _borrow_ the test for this one? 
https://github.com/sindresorhus/escape-string-regexp/blob/master/test.js#L1

Reply:
Done. "Borrowed".